pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -488,6 +489,11 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 			r.status.SetDegraded("Failed to get curator credentials", err.Error())
 			return reconcile.Result{}, err
 		}
+		applyTrial, err = r.applyElasticTrialSecret(ctx, install)
```

Comment:
What does this have to do with FIPS mode?

Reply:
We will leverage a FIPS build of elasticsearch, which requires a license.

Reply:
I added a comment to the method itself to detail why.